Sources/Purchasing/VirtualCurrencyMetadata.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  VirtualCurrencyInfo.swift
+//
+//  Created by RevenueCat on 3/19/24.
+
+import Foundation
+
+/**
+ * Information about a virtual currency available in an offering.
+ */
+@objc(RCVirtualCurrencyMetadata) public final class VirtualCurrencyMetadata: NSObject {
+
+    /**
+     * The name of the virtual currency.
+     */
+    @objc public let name: String
+
+    /**
+     * The description of the virtual currency.
+     */
+    @objc public let currencyDescription: String
```

Comment:
Since this class is going to be public, we might want to consider renaming this to `serverDescription` to match the naming convention used in `VirtualCurrency` from the balance fetching PR